src/slim-select/slim.ts

```diff
@@ -293,6 +293,8 @@ export class Slim {
     text.innerHTML = (optionObj.innerHTML && this.main.config.valuesUseText !== true ? optionObj.innerHTML : optionObj.text)
     value.appendChild(text)
 
+    if (optionObj.mandatory) { return value }
```

Comment:
Can you wrap the delete span instead of returning. Just in case additional features might want to add code at a later point.